config.py

```diff
@@ -910,30 +916,49 @@
     ```
 """
 
-contrasts: Iterable[Tuple[str, str]] = []
+contrasts: Iterable[Union[Tuple[str, str], ArbitraryContrast]] = []
```

Comment:
Let's do it like this until we drop support for Python <3.10 and can replace `Union` with `|`

```suggestion
contrasts: Iterable[
    Union[
        Tuple[str, str],
        ArbitraryContrast
    ]
] = []
```